project1/backend/models/queueModel.js

```diff
@@ -0,0 +1,92 @@
+const AppError = require('../utils/AppError');
+const formatName = require('../utils/formatName');
+const decoratorTTL = require('../utils/decoratorTTL');
+const Ajv = require('ajv');
+
+const ajv = new Ajv();
+
+class QueueModel {
```

Comment:
Why QueueModel? Why is not PatientsModel?

Reply:
We have discussed during lectures that we have Queue and Resolutions API's, and in the future, we might need Patients as a separate API that ties everything together, but for this specific project, it is not required.